server/controllers/personalInfo.js

```diff
@@ -0,0 +1,21 @@
+const PersonalInfo = require('../models/PersonalInfo');
+const asyncHandler = require("express-async-handler");
+
+exports.createPersonalInfo = asyncHandler(async (req, res) => {
```

Comment:
Looks good for a start. However, under the current scenario, won't be creating a new document each time a user submits his profile info?